### PR TITLE
Move appkey to strings.xml

### PR DESCRIPTION
**Pull request by qinweigong**
@aaustin 

- [ ] Update dashboard on-boarding steps

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Branch Android SDK change log 
+
+- v1.2.9: Moved app key to strings.xml; Added constants for OG tags and redirect URLs
+
+- v1.2.8: Fixed close issue due to rotation
```
Comment (aaustin) [inline ChangeLog.md]: Nice! You added it for me. Thanks man

**Comment by aaustin** [comment]
@SleeplessByte How does one leverage the /debug and /release strings file? Where do you toggle? Is that an Android studio think?

**Comment by aaustin** [comment]
we're going to do a revamp of debug/live management in the next month and I want to make sure to include your suggestion in the update. By the way :+1:

**Comment by SleeplessByte** [comment]
@aaustin it's a `gradle` build thing. I don't know if eclipse supports this, but if I remember correctly, this is the way to do it.

**Pull request merged**